force-app/main/default/classes/functions/AsyncFunctionInvocationQueueable.cls

```diff
@@ -74,6 +74,10 @@ public class AsyncFunctionInvocationQueueable implements Queueable, Database.All
 
     /**
      * Finalizer implementation captures failed function invocation jobs on unexpected execptions, eg limit errors.
+     *
+     * Requires that the invoking user is assigned access to AsyncFunctionInvocationRequest__c (likely via
```

Comment:
double-checked and the README includes that information - so requirement already doc'd
> You must assign Edit access to AsyncFunctionInvocationRequest__c for invoking users. You can do so either with the FunctionReference.PermissionSetOrGroup__c‘s permission set or another permission set.